### DISCUSSION TITLE
EIN (00-0000000 or 0-0000000) representation

### DISCUSSION THREAD

**Discussion by smblee** [uNmAnNeR/imaskjs#711] [Q&A]
This might be a noob question but im having hard time getting EIN input to work. I am trying to allow either `00-0000000` or `0-0000000` to work (with dash automatically added), but the latter is never getting picked up, unless I explicitly draw the dash. 

Naive way of thinking about this would be `mask: '0[0]-0000000'`.

Ideal way would be getting something like this following regex to work for a very strict EIN check (https://www.regextester.com/108957), but its not necessary.

**Comment by uNmAnNeR** [comment]
you can use dynamic mask
